src/codemodder/cli.py

```diff
@@ -144,13 +145,13 @@ def parse_args(argv, codemod_registry):
     parser.add_argument(
         "--path-exclude",
         action=CsvListAction,
-        default="",
+        default=DEFAULT_EXCLUDED_PATHS,
         help="Comma-separated set of UNIX glob patterns to exclude",
```

Comment:
just noticing from my semgrep PR that we need the defaults exclude/include to always be lists, which you've done in this PR. Pointing that out since the help str indicates command sep which is what the user does, then gets converted to a list and we always assume it's a list.

Reply:
Yup I added some type annotations to clarify that for the `code_directory` API.